### PR TITLE
Print filename in "there are pending proofs" error

### LINKED ISSUE #9404: "There are pending proofs" message does not mention the file
Compiling a file which contains:

```coq
Goal True.
```

with coqc produces the following error message:
```coq
Error: There are pending proofs: Unnamed_thm
```

which does not say which file the error comes from.

### PR DESCRIPTION
**Pull request by SkySkimmer**
Fix #9404

### CODE REVIEW

**Comment by Alizter** [comment]
Can you add a period?

**Comment by SkySkimmer** [comment]
There's already one

**Review by Alizter — APPROVED** [review]
That line was truncated on my phone sorry.

**Comment by Alizter** [comment]
Will merge later today

File: toplevel/ccompile.ml [hunk 1]
```diff
@@ -95,7 +95,7 @@ let create_empty_file filename =
 let check_pending_proofs filename =
   let pfs = Vernacstate.Declare.get_all_proof_names () [@ocaml.warning "-3"] in
   if not (CList.is_empty pfs) then
-    fatal_error (str "There are pending proofs: "
+    fatal_error (str "There are pending proofs in file " ++ str filename ++ str": "
```
Comment (ejgallego) [inline toplevel/ccompile.ml:98]: Maybe we should take a location?

Comment (SkySkimmer) [reply toplevel/ccompile.ml:98]: I thought of using locations but then either we say the error is at line 1 or we have to do something complicated to get the last line (which probably isn't much better in the end)

Comment (ejgallego) [reply toplevel/ccompile.ml:98]: We could use the location of the last command seen, but indeed, not sure about it.

I have a patch in serapi / jscoq that actually maps list of vernacs to something with structure, in this case we know when sections / proofs / etc... are open, so we could try to track the proof that is actually to blame, but indeed doing that right seems like some work ahead.

**Review by ejgallego — APPROVED** [review]

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**